Unix/pal/alloc.c

```diff
@@ -124,7 +124,7 @@ void PAL_DumpAllocList()
 
     for (p = _list; p; p = p->next)
     {
-        trace_DumpAllocList_Block(p->file, (int)p->line, p, p->magic, p->id, p->size);
+        trace_DumpAllocList_Block(p->file, (int)p->line, p+1, p->magic, p->id, p->size);
```

Comment:
Um, why `p+1`? Minimally, this needs a comment. But it looks like an error, explain please.

Reply:
Added comment to explain +1.